src/common/utils/create_entity.py

```diff
@@ -38,14 +38,14 @@ def parse_value(attr: BlueprintAttribute, blueprint_provider):
         default_value = attr.default
         type = attr.attribute_type
 
-        if default_value is not None and len(str(default_value)) > 0 and attr.is_array:
+        if default_value is not None and len(str(default_value)) > 0 and attr.is_array and type not in PRIMITIVES:
             try:
                 return json.loads(default_value)
             except JSONDecodeError:
                 print(f"invalid default value: {default_value} for attribute: {attr}")
                 return []
 
-        if default_value == "":
+        if default_value is None:
```

Comment:
this change was necessary because the type of "default" in the BlueprintAttribute python class changed from string to `StrictStr | StrictInt | StrictFloat | StrictBool | list | dict | None`